src/bioregistry/data/bioregistry.json

```diff
@@ -14623,6 +14623,38 @@
       "uri_format": "https://www.cellrepo.com/repositories/$1"
     }
   },
+  "cellxgene": {
+    "contributor": {
+      "email": "robert.j.allaway@gmail.com",
+      "github": "allaway",
+      "name": "Robert Allaway",
+      "orcid": "0000-0003-3573-3565"
+    },
+    "description": "CELLxGENE is an interactive data visualization and exploration tool developed by the Chan Zuckerberg Initiative that enables researchers to analyze and share single-cell genomics datasets. It provides a user-friendly interface for biologists and computational scientists to interrogate gene expression patterns across different cell types. Identifiers correspond to collections made available in CELLxGENE.",
```

Comment:
let's try and update the description to say what kinds of entities this semantic space covers. From https://cellxgene.cziscience.com/collections, this appears to cover collections of datasets.

However, https://cellxgene.cziscience.com/datasets also appears to use the same URI format for resolution. Maybe you can help understand better if these are a unified semantic space or two different ones

Reply:
I specifically added "Identifiers correspond to collections made available in CELLxGENE." to the description.

Reply:
ahh thanks, that's hiding at the end. It's better not to bury the lede here, I will try and update the https://github.com/biopragmatics/bioregistry/blob/main/docs/CONTRIBUTING.md#writing-a-good-description guide accordingly

Reply:
@cthoyt - my understanding is that these are two semantic spaces. I was initially going to suggest "datasets" as the primary one, but as there are no landing pages for datasets, only collections, I figured that was perhaps preferable. I think `cellxgene.dataset` would be a valuable addition tho.